melos.yaml

```diff
@@ -113,6 +113,9 @@ scripts:
         "flutter test --no-pub --platform=chrome"
     description: Run `flutter test --platform=chrome` for a specific '*web' package.
     select-package:
+      ignore:
+        # TODO(Salakar): Package has no web tests.
```

Comment:
nit but I'd add an empty test instead

To not forget to uncomment this. Happened to me before :singe_rien_vu:

Reply:
Fair point, will add an empty test :)

Reply:
Done!